examples/demo-rollup/src/main.rs

```diff
@@ -24,15 +27,24 @@ pub fn initialize_ledger(path: impl AsRef<std::path::Path>) -> LedgerDB {
     ledger_db
 }
 
+async fn start_rpc_server(module: RpcModule<()>, address: SocketAddr) {
+    let server = jsonrpsee::server::ServerBuilder::default()
+        .build([address].as_ref())
+        .await
+        .unwrap();
+    let _server_handle = server.start(module).unwrap();
+    futures::future::pending::<()>().await;
+}
+
 pub fn get_genesis_config() -> GenesisConfig<DefaultContext> {
     let sequencer_private_key = DefaultPrivateKey::generate();
     create_demo_genesis_config(
         100000000,
         sequencer_private_key.default_address(),
         vec![
-            99, 101, 108, 101, 115, 116, 105, 97, 49, 122, 102, 118, 114, 114, 102, 97, 113, 57,
-            117, 100, 54, 103, 57, 116, 52, 107, 122, 109, 115, 108, 112, 102, 50, 52, 121, 115,
-            97, 120, 113, 102, 110, 122, 101, 101, 53, 119, 57,
+            99, 101, 108, 101, 115, 116, 105, 97, 49, 113, 112, 48, 57, 121, 115, 121, 103, 99,
```

Comment:
I know this was just an update, but it would be nice to define this magic vector somewhere and then use its variable for readability, so that first time readers would know what it means.

Reply:
just checked with preston - since its consensus critical /  part of the genesis config, its better to have it in the codebase as opposed to runtime configuration. moved it back into the code for now

Reply:
Good call on the magic value though - we should add a doc explaining how and why to pick this value (and the magic 1000000).

Reply:
added sequencer_da_address as a constant